[2025-01-21 13:45:39,261][exploit_info.py:101][ERROR] Error: Index(...) must be called with a collection of some kind, 'Node_raw' was passed
[2025-01-21 14:39:57,026][exploit_info.py:101][ERROR] Error: Index(...) must be called with a collection of some kind, 'Node_raw' was passed
[2025-01-21 15:15:53,984][exploit_info.py:222][ERROR] Error: 'target'
[2025-01-21 15:15:53,985][exploit_info.py:275][ERROR] Error: 'target'
[2025-01-21 15:15:53,985][exploit_info.py:275][ERROR] Error: 'target'
